seeds/park-seeds.js

```diff
@@ -42,21 +42,125 @@ const generateSqlArray = (parkData) => {
   let formattedSql = [];
   let parks = parkData.parks;
   for (let x = 0; x < parks.length; x++) {
-    formattedSql.push({id: x + 1, park_id: parks[x].id})
+    formattedSql.push({
+      // id: x + 1, 
+      park_id: parks[x].id, 
+      park_code: parks[x].parkCode,
+      name: parks[x].fullName, 
+      description: parks[x].description,
+      state: parks[x].states,
+      url: parks[x].url,
+      image: parks[x].images[0].url,
+      latitude: parks[x].latitude,
+      longitude: parks[x].longitude
+    })
   }
   return formattedSql;
 }
 
+const generateCategoriesSqlArray = (parkData) => {
+  let categorySql = [];
+  let parks = parkData.parks;
+  console.log(parks.length);
+  for (let i = 0; i < parks.length; i++) {
+    let activities = parks[i].activities;
+    if (activities.length > 0) {
+      for (let j = 0; j < activities.length; j++) {
+        if (activities[j].categoryAbbr) {
+          categorySql.push({
+            name: parks[i].fullName, 
+            category_abbr: activities[j].categoryAbbr
+          })
+        }
+      }
+    }
+  }
+  console.log(categorySql);
+  return categorySql;
+}
+
 const seedParks = async () => {
   const parkData = await getParkData();
-
-  writeToFile("./data/parks.json", JSON.stringify(parkData))

```

Comment:
Since I've pulled all fields needed for the parks, I think we can delete the fn that creates the json file. Maybe just comment out until more confident though,